src/Infrastructure/LeanCode.ExternalIdentityProviders/ExternalIdentityProvidersModule.cs

```diff
@@ -2,28 +2,76 @@
 using Autofac;
 using LeanCode.Components;
 using LeanCode.ExternalIdentityProviders.Apple;
+using LeanCode.ExternalIdentityProviders.Facebook;
 using LeanCode.ExternalIdentityProviders.Google;
 using Microsoft.AspNetCore.Identity;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace LeanCode.ExternalIdentityProviders
 {
+    [Flags]
+    public enum Providers
+    {
+        Facebook = 0b001,
+        Apple = 0b010,
+        Google = 0b100,
+
+        None = 0b000,
+        All = 0b111,
+    }
+
     public class ExternalIdentityProvidersModule<TUser> : AppModule
         where TUser : IdentityUser<Guid>
     {
+        private readonly Providers configuration = Providers.All;
```

Comment:
Default value is unnecessary now that there is no parameterless constructor.
```suggestion
        private readonly Providers configuration;
```